content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,155 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+### Attaching Files
+
+Attaching files to input elements or dropping them over the page is available
+in Cypress 10.0. Read the [`.attachFile()` API docs](/api/commands/attachfile)
+for more information on how this works and how to use it.
+
+The
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload)
+plugin has been deprecated in favor of `.attachFile()` built into Cypress.
+There's guidance below on how to migrate from the
+[`cypress-file-upload`](https://github.com/abramenal/cypress-file-upload)
+plugin to Cypress's built-in attachFile command.
+
+#### Quick guide
+
+The argument signature is different for Cypress' builtin `.attachFile()` command than the version the
+`cypress-file-upload` plugin provided. You can follow the steps below for each argument in order to migrate:
+
+In the first argument:
+- `filePath`: Prefix value with `fixtures/`. Rename to `contents`.
+- `fileContent`: Use `Buffer.from()` rather than `Cypress.Blob` methods. Rename to `contents`.
+- `encoding`: Remove. No longer needed due to improved binary file handling in Cypress 9.0.
+- `mimeType`: Remove. No longer needed due to improved binary file handling in Cypress 9.0.
+
+In the second argument:
+- `subjectType`: Rename to `action`.
+- `allowEmpty`: Remove. `.attachFile()` does not check the length of a file read from disk, only its existence.
+
+### Examples
+
+Below are several examples of migrating various commands from `cypress-file-upload`
+to the builtin `.attachFile()`.
+
+#### Read and attach a fixture
+
+<Badge type="danger">Before</Badge> Attaching a fixture from disk with
+`cypress-file-upload`
+
+```js
+cy.get('[data-cy="file-input"]').attachFile('myfixture.json');
+```
+
+<Badge type="success">After</Badge> Attaching a fixture from disk with
+Cypress 10.0. Cypress follows paths from your project root
+(same as [`cy.readFile()`](/api/commands/readfile)).
+
+
+```js
+cy.get('[data-cy="file-input"]').attachFile('cypress/fixtures/myfixture.json');
```

Comment:
Given the note
> `filePath`: Prefix value with `fixtures/`. Rename to `contents`.

Should this example be:
```js
cy.get('[data-cy="file-input"]').attachFile('fixtures/myfixture.json');
```

Reply:
This section is correct, I've updated the quick guide to match.